src/__tests__/buildTransfers.test.ts

```diff
@@ -0,0 +1,188 @@
+import { expect } from "chai";
+import { buildTransfers, TEN } from "../transfers";
+import { SafeInfo, Transaction } from "@gnosis.pm/safe-apps-sdk";
+import { TokenInfo } from "@uniswap/token-lists";
+import BigNumber from "bignumber.js";
+import { Payment } from "../components/CSVForm";
+import { fetchTokenList, TokenMap } from "src/tokenList";
+import { utils, Bytes } from "ethers";
+
+let dummySafeInfo: SafeInfo = {
+  safeAddress: "0x123",
+  network: "rinkeby",
+  ethBalance: "100",
+};
+
+let tokenList: TokenMap;
+let listedTokens: string[];
+let listedToken: TokenInfo;
+const receiverAddress = "0x1000000000000000000000000000000000000000";
+const unlistedToken: TokenInfo = {
+  address: "0x6b175474e89094c44da98b954eedeac495271d0f",
+  decimals: 18,
+  symbol: "UNL",
+  name: "Unlisted",
+  chainId: -1,
+};
+
+// TODO - make method erc20TransferData and replace data checks with this instead of hardcoded strings.
+// function erc20TransferData(amount: BigNumber, receiver: string): Bytes {}
+
+describe("Build Transfers:", () => {
+  beforeEach(async () => {
+    tokenList = await fetchTokenList(dummySafeInfo.network);
+    listedTokens = Array.from(tokenList.keys());
+    listedToken = tokenList.get(listedTokens[0]);
+  });
+
+  function listedUnlistedAndNativePayments(
+    amount,
+    receiver
+  ): [Payment, Payment, Payment] {
+    return [
+      // Listed ERC20
+      {
+        receiver,
+        amount: amount,
+        tokenAddress: listedToken.address,
+        decimals: listedToken.decimals,
+      },
+      // Unlisted ERC20
+      {
+        receiver,
+        amount: amount,
+        tokenAddress: unlistedToken.address,
+        decimals: unlistedToken.decimals,
+      },
+      // Native Asset
+      {
+        receiver,
+        amount: amount,
+        tokenAddress: null,
+        decimals: null,
+      },
+    ];
+  }
+
+  describe("Integers", () => {
+    it("works with large integers on listed, unlisted and native asset transfers", () => {
+      let max_amount = new BigNumber(2 ** 255 - 1).dividedBy(
+        TEN.pow(listedToken.decimals)
+      );
+      let fractional_amount = new BigNumber("0.0000001");
```

Comment:
The fractional amount is not needed in this test method

Reply:
Why not?

Reply:
Because its the "Integers" Testcase which only uses `max_amount`.
`fractional_amount` is then again defined in the Decimals Case and also used there.

Reply:
Oh right.. another copy-pasta mistake. I think I've been staring at this too long. Need to stand up for a few minutes.

Reply:
I would expect our linter to catch `no-unused-variables`